1.7.0 build failure on macOS configure: error: Required function not present

```
==> ./autogen.sh
+ type autoconf
+ type automake
+ type aclocal
+ aclocal
+ autoconf
+ automake --add-missing
configure.ac:10: installing './compile'
configure.ac:16: installing './config.guess'
configure.ac:16: installing './config.sub'
configure.ac:7: installing './install-sh'
configure.ac:7: installing './missing'
Makefile.am: installing './depcomp'
+ echo 'now you can run ./configure && make to build openfortivpn'
now you can run ./configure && make to build openfortivpn
==> ./configure --disable-dependency-tracking --disable-silent-rules --prefix=/usr/local/Cellar/openfortivpn/1.7.0
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... ./install-sh -c -d
checking for gawk... no
checking for mawk... no
checking for nawk... no
checking for awk... awk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking for gcc... clang
[...]
checking for printf... yes
checking for pthread_cancel... yes
checking for pthread_cond_init... yes
checking for pthread_cond_signal... yes
checking for pthread_cond_wait... yes
checking for pthread_join... yes
checking for pthread_mutexattr_init... yes
checking for pthread_mutexattr_setrobust... no
configure: error: Required function not present
```

--- a/configure.ac
+++ b/configure.ac
@@ -38,7 +38,7 @@ AC_TYPE_UINT8_T
 # Checks for library functions.
 AC_FUNC_MALLOC
 AC_FUNC_REALLOC
-AC_CHECK_FUNCS([atoi close connect execv exit _exit fclose fcntl fflush fopen forkpty fprintf fputs free freeaddrinfo freeifaddrs freopen fwrite getaddrinfo getchar getenv getopt_long htons index inet_addr inet_ntoa isatty malloc memcpy memmem memmove memset ntohs open openlog pclose popen printf pthread_cancel pthread_cond_init pthread_cond_signal pthread_cond_wait pthread_join pthread_mutexattr_init pthread_mutexattr_setrobust pthread_mutex_destroy pthread_mutex_init pthread_mutex_lock pthread_mutex_unlock pthread_sigmask puts read realloc rewind select setenv sigaddset sigemptyset signal snprintf socket sprintf strcasestr strcat strchr strcmp strcpy strdup strerror strlen strncasecmp strncat strncpy strsignal strstr strtok strtok_r strtol syslog system tcsetattr usleep vprintf vsnprintf vsyslog write], [], AC_MSG_ERROR([Required function not present]))
-AC_CHECK_FUNCS([X509_check_host])
+AC_CHECK_FUNCS([atoi close connect execv exit _exit fclose fcntl fflush fopen forkpty fprintf fputs free freeaddrinfo freeifaddrs freopen fwrite getaddrinfo getchar getenv getopt_long htons index inet_addr inet_ntoa isatty malloc memcpy memmem memmove memset ntohs open openlog pclose popen printf pthread_cancel pthread_cond_init pthread_cond_signal pthread_cond_wait pthread_join pthread_mutexattr_init pthread_mutex_destroy pthread_mutex_init pthread_mutex_lock pthread_mutex_unlock pthread_sigmask puts read realloc rewind select setenv sigaddset sigemptyset signal snprintf socket sprintf strcasestr strcat strchr strcmp strcpy strdup strerror strlen strncasecmp strncat strncpy strsignal strstr strtok strtok_r strtol syslog system tcsetattr usleep vprintf vsnprintf vsyslog write], [], AC_MSG_ERROR([Required function not present]))
+AC_CHECK_FUNCS([pthread_mutexattr_setrobust X509_check_host])
 
 AC_OUTPUT(Makefile)

--- a/src/log.c
+++ b/src/log.c
@@ -55,7 +55,7 @@ void init_logging(void)
 	is_a_tty = isatty(STDOUT_FILENO);
 
 	pthread_mutexattr_init(&mutexattr);
-#ifndef __APPLE__
+#ifdef HAVE_PTHREAD_MUTEXATTR_SETROBUST
 	pthread_mutexattr_setrobust(&mutexattr, PTHREAD_MUTEX_ROBUST);
 #endif
 	pthread_mutex_init(&mutex, &mutexattr);

